src/components/secure-backup/container.tsx

```diff
@@ -35,11 +52,13 @@ export class Container extends React.Component<Properties> {
       recoveryKey: generatedRecoveryKey || '',
       successMessage,
       errorMessage,
+      assetsPath: config.assetsPath,
```

Comment:
this will need updating to `config.videoAssetsPath` once env vars PRs merged

Reply:
If you have PR dependencies you should base this branch on the branches that are required. That way it's very difficult to forget stuff like this.